python_scripts/trees.py

```diff
@@ -182,13 +183,13 @@ def plot_decision_function(X, y, clf, ax=None):
 )
 
 # %% [markdown]
-# Indeed, it is not a surprise. We earlier saw that a single feature will not
-# help separating the three species. However, from the previous analysis we
-# saw that using both features should be useful to get fairly good results.
+# Indeed, it is not a surprise. We saw earlier that a single feature will not
+# be able to separate all three species. However, from the previous analysis we
+# saw that by using both features we should be able to get fairly good results.
 # Considering the mechanism of the decision tree illustrated above, we should
-# repeat the partitioning on each rectangle that was previously created. In
-# this regard, we expect that the partition will be using the feature "Culmen
-# Depth" this time.
+# repeat the partitioning on the resulting rectangle created by the first
+# split. In this regard, we expect that the second partition will be using
```

Comment:
Might be a little confusing, the second partitioning sounds like there will be one 2nd partition. But there will be 2 partitions? One each for the each 'resulting rectangle' ?

Reply:
https://github.com/lucyleeow/scikit-learn-mooc/pull/3